Web/tues_pairs/src/components/App/index.jsx

```diff
@@ -22,8 +22,8 @@ import LandingPageBase from '../Pages/Landing/landing';
 const App = () => (
     <Router>
         <Navigation />
+        <Route exact path={ROUTES.LANDING} component={LandingPageBase} />
         <div className="main">
-            <Route exact path={ROUTES.LANDING} component={LandingPageBase} />
```

Comment:
What difference does this make?

Reply:
I needed that for the design (css). I differentiate my main pages with this class="main". For the landing page I used background image and it's css was having some issues with this <div className="main">, so i extracted it outside, in order to have a responsive landing page.